test/live-then-grow.js

```diff
@@ -0,0 +1,61 @@
+const test = require('tape')
+const fs = require('fs')
+const validate = require('ssb-validate')
+const ssbKeys = require('ssb-keys')
+const path = require('path')
+const FlumeLog = require('async-flumelog')
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+const pull = require('pull-stream')
+const { addMsg } = require('./common')()
+const jitdb = require('../index')
+const {
+  and,
+  query,
+  fromDB,
+  live,
+  toPullStream,
+  slowEqual,
+} = require('../operators')
+
+const dir = '/tmp/jitdb-live-then-grow'
+rimraf.sync(dir)
+mkdirp.sync(dir)
+
+var keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+
+test('live from empty log', (t) => {
```

Comment:
I can update this to use prepareAndRun, I actually didn't use it here because I thought I would need to close jitdb and reopen it, but it turned out I didn't need to do that.

but i'll do this update tomorrow, unless you want to do it

Reply:
Ok, this looks good. I'll wait for the update and merge